### ISSUE TITLE
Shouldn't allow title of proposal to be changed after last date of submission

### ISSUE THREAD

**Issue by vnbang2003** [pythonindia/junction#498]
We should not allowed proposal title to be changed after the last date of submission.
If reviewers suggest for change it can just be done by CFP coordinator or admin.

**Comment by ananyo2012** [comment]
@sayanchowdhury @anistark Do you think this is needed in a proposal ?

**Comment by palnabarun** [comment]
@ananyo2012 Yes. This is needed. After the submission dates are over, it is unfair to let people freely change the title/content of the talk. At least if the title becomes immutable after the deadline, the content can't be freely changed to whatever the proposer wants.
